tutorials/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -35,7 +35,7 @@ Here is a complete class example based on these guidelines:
 
     signal state_changed(previous, new)
 
-    export var initial_state = NodePath()
+    @export var initial_state := NodePath()
```

Comment:
We should remove this example and export the node directly.

Reply:
Updated to use a Node directly